.github/workflows/run_psi4_test.yml

```diff
@@ -23,11 +24,12 @@ jobs:
     - run: conda list
     - run: conda config --show
 
-    - name: Install psi4
+    - name: Install psi4 ${{ matrix.python-version }}
       shell: bash -el {0}
       run: |
-        conda install psi4 python=3.8 -c psi4
+        conda install psi4 python=${{ matrix.python-version }} -c psi4
```

Comment:
Changing this line to `conda install psi4 python=${{ matrix.python-version }} -c conda-forge/label/libint_dev -c conda-forge` may work better.

Reply:
This worked, you're rad